launcher.sh

```diff
@@ -3,7 +3,7 @@ WD=`dirname "$0"`
 EVBIN="Ev Nova"
 
 export DYLD_FORCE_FLAT_NAMESPACE=1
-export DYLD_INSERT_LIBRARIES="$WD/libNova.A.dylib"
+export DYLD_INSERT_LIBRARIES="$WD/libNova.A.dylib:/System/Library/Frameworks/OpenGL.framework/Resources/GLEngine.bundle/GLEngine"
```

Comment:
Wrap the additional library in an if -e, to only append when it's verified its presence in that location, and I'll merge it in.